### PR TITLE
[Improve][Docker]Modified Spark and Flink starter Dockerfiles

### PR DESCRIPTION
**Pull request by arifgore**
## Check list

* [X] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-core/seatunnel-flink-starter/src/main/docker/Dockerfile [hunk 1]
```diff
@@ -20,18 +20,29 @@ FROM $BASE_IMAGE
...
 ARG FLINK_VERSION
+ARG SEATUNNEL_VERSION
 
 RUN mkdir -p /flink ; cd /flink ; \
     tar_file=flink-${FLINK_VERSION}-bin-scala_${SCALA_VERSION}.tgz ; \
     curl -LsO https://archive.apache.org/dist/flink/flink-${FLINK_VERSION}/$tar_file ; \
     tar -zxf $tar_file --strip 1 -C . ; \
     rm $tar_file
-
+
```
Comment (hailin0) [inline seatunnel-core/seatunnel-flink-starter/src/main/docker/Dockerfile:30]: revert？

File: seatunnel-core/seatunnel-spark-starter/src/main/docker/Dockerfile [hunk 2]
```diff
@@ -29,9 +30,14 @@ RUN mkdir -p /spark ; cd /spark ; \
...
+    tar -zxf $seatunnel_archive_name --strip 1 -C . ; \
+    rm $seatunnel_archive_name; \
+    rm -rf /seatunnel/connectors/flink* /seatunnel/bin/start-seatunnel-{"flink","sql"}.sh /seatunnel/lib/seatunnel-core-flink* /seatunnel/config/seatunnel-env.sh
+
 
-COPY target/seatunnel-core-spark.jar /seatunnel/lib/
-COPY src/main/bin /seatunnel/bin/
+WORKDIR /seatunnel
 
 ENTRYPOINT [ "/seatunnel/bin/start-seatunnel-spark.sh" ]
```
Comment (hailin0) [inline seatunnel-core/seatunnel-spark-starter/src/main/docker/Dockerfile:43]: Need to update? same as flink

**Review by hailin0 — COMMENTED** [review]
great

**Comment by EricJoy2048** [comment]
Please resolve conflicts.

File: seatunnel-core/seatunnel-flink-starter/src/main/docker/Dockerfile (hunk 1 above)
Comment (arifgore) [reply seatunnel-core/seatunnel-flink-starter/src/main/docker/Dockerfile:30]: I don't get it. What should I revert exactly?

**Pull request closed without merging**